Bio/GenBank/Scanner.py

```diff
@@ -1177,6 +1177,16 @@ def _feed_first_line(self, consumer, line):
         the column based layout.
 
         We also try to cope with GenBank like files with partial LOCUS lines.
+
+        As of release 229.0, the columns are no longer strictly in a given
+        position. See:
+
+        Historically, the LOCUS line has had a fixed length and its elements
```

Comment:
This doesn't flow nicely from the previous paragraph. Is this a quote from the NCBI? If so putting opening and closing quote marks round it would be enough.

You could explore making it a nice block quote when rendered as RST using http://rst.ninjs.org/ or similar but that isn't required.